hedera-node/configuration/dev/application.properties

```diff
@@ -35,3 +35,4 @@ tss.historyEnabled=false
 contracts.throttle.throttleByGas=true
 contracts.opsDurationThrottleCapacity=500000000000000
 contracts.opsDurationThrottleUnitsFreedPerSecond=500000000000000
+nodes.feeCollectionAccountEnabled=true
```

Comment:
shouldn't this be false?

Reply:
No I have enabled this feature only in our CI, so in `dev/application.properties`. Once we are sure this will go out in 0.70, I can enable it on main and release/0.70